lib/Conversion/TritonGPUToLLVM/HistogramOpToLLVM.cpp

```diff
@@ -80,6 +81,11 @@ static SmallVector<Value> computeWarpLevelHistogram(
       warpLevelHistogram[k] = b.add(warpLevelHistogram[k], bitCount);
     }
   }
+  for (auto i = 0; i < warpLevelHistogram.size(); ++i) {
+    warpLevelHistogram[i] =
+        b.sdiv(warpLevelHistogram[i], b.i32_val(replicationFactor));
```

Comment:
unsigned div should be more efficient for power of 2

Reply:
I checked and this lowers to lshr. I assume LLVM can fix it because this value should be known to be nonnegative